src/itest/kotlin/nl/lifely/zac/itest/ZakenRESTServiceTest.kt

```diff
@@ -129,7 +131,7 @@ class ZakenRESTServiceTest : BehaviorSpec({
         }
     }
     Given("A zaak has been created") {
-        When("the assign to zaak endpoint is called with a group") {
+        When("the 'assign to zaak' endpoint is called with a group") {
             val response = itestHttpClient.performPatchRequest(
                 url = "$ZAC_API_URI/zaken/toekennen",
                 requestBodyAsString = "{\n" +
```

Comment:
json doesn't care about newlines, do we need to have them here?  Also: Kotlin has multi-line strings I would prefer those instead of string concatenation syntax. Both minor, won't stop my approval.

Reply:
I sometimes like to add newlines in our JSON because I typically run WildFly using debug log level settings so that I can see all the HTTP traffic (and much more) in the logs including the JSON in our requests. Having newlines in the JSON makes it a lot easier to read and debug what's going on I find.

Reply:
So yes, JSON does not care but I do. :-)

but maybe there is a better solution than us having to explicitly add newlines.. pretty formatting of JSON is standard practise of course